include/inexor/vulkan-renderer/meta.hpp.in

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <array>
+
+namespace inexor::vulkan_renderer {
+
+/// The following data will be replaced by CMake setup.
+static constexpr char m_application_name[] = "${APP_NAME}";
```

Comment:
These shouldn't be `m_` since they're global. Probably should be `APPLICATION_NAME` like other constants? Also `static` doesn't do anything here, and the strings should be declared as `const char *name` instead of `char name[]`.

Reply:
Done.